lucille-examples/lucille-distributed-example/src/test/java/com/kmwllc/lucille/distributed/VerifyIngestResult.java

```diff
@@ -27,30 +27,46 @@ public void verifyIngestResult() throws Exception {
       docs.add(json.get("response").get("docs").get(i));
     }
 
-    assertTrue(docs.stream()
-        .anyMatch(doc -> doc.get("id").asText().equals("source.csv-1")
-            && doc.get("source").get(0).asText().equals("/conf/source.csv")
-            && doc.get("filename").get(0).asText().equals("source.csv") && doc.get("my_country").get(0).asText().equals("Italy")
-            && doc.get("my_price").get(0).asInt() == 30 && doc.get("my_name").get(0).asText().equals("Carbonara")));
-    assertTrue(docs.stream()
-        .anyMatch(doc -> doc.get("id").asText().equals("source.csv-2")
-            && doc.get("source").get(0).asText().equals("/conf/source.csv")
-            && doc.get("filename").get(0).asText().equals("source.csv") && doc.get("my_country").get(0).asText().equals("Italy")
-            && doc.get("my_price").get(0).asInt() == 10 && doc.get("my_name").get(0).asText().equals("Pizza")));
-    assertTrue(docs.stream()
-        .anyMatch(doc -> doc.get("id").asText().equals("source.csv-3")
-            && doc.get("source").get(0).asText().equals("/conf/source.csv")
-            && doc.get("filename").get(0).asText().equals("source.csv") && doc.get("my_country").get(0).asText().equals("Korea")
-            && doc.get("my_price").get(0).asInt() == 12 && doc.get("my_name").get(0).asText().equals("Tofu Soup")));
-
-    assertTrue(
-        docs.stream().anyMatch(doc -> doc.get("id").asText().equals("3") && doc.get("my_country").get(0).asText().equals("USA")
-            && doc.get("my_price").get(0).asInt() == 30 && doc.get("my_name").get(0).asText().equals("Burger")));
-    assertTrue(
-        docs.stream().anyMatch(doc -> doc.get("id").asText().equals("4") && doc.get("my_country").get(0).asText().equals("USA")
-            && doc.get("my_price").get(0).asInt() == 10 && doc.get("my_name").get(0).asText().equals("Salad")));
-    assertTrue(
-        docs.stream().anyMatch(doc -> doc.get("id").asText().equals("5") && doc.get("my_country").get(0).asText().equals("France")
-            && doc.get("my_price").get(0).asInt() == 12 && doc.get("my_name").get(0).asText().equals("Wine")));
+    JsonNode csvDoc1 = docs.get(0);
```

Comment:
can we be sure the docs will always appear in dest.json in this exact order?

dest.json is created via a solr query http://solr:8983/solr/quickstart/query?q=*:*'

with a "*:*" query without a sort param, the docs will be returned in the order indexed

if we have multiple worker threads its conceivable that the docs will be indexed in a different order from what's expected here

could add a sort to the query (on the id) to be explicit about the order we want, but that would change the order from what the test is currently expecting

Reply:
could also sort the docs here in this Junit test code before checking their values... probably easier to do it this way